src/main.cpp

```diff
@@ -15,16 +15,27 @@ int main(int argc, char** argv) {
         RIP("XDG_RUNTIME_DIR not set!");
 
     // parse some args
-    for (int i = 1; i < argc; ++i) {
-        if (!strcmp(argv[i], "--i-am-really-stupid"))
-            ignoreSudo = true;
-    }
+    if (!strcmp(argv[1], "i-am-really-stupid"))
+        ignoreSudo = true;
+    else if (!strcmp(argv[1], "help"))
+        std::cout << "Available commands:\n";
+        std::cout << "  help                 - show this help text\n";
+        std::cout << "  version              - show version\n";
+        std::cout << "  --i-am-really-stupid - ignore sudo run check\n";
+        return 0;
+    else if (!strcmp(argv[1], "vesrsion"))
+        std::cout << "V5.0\n";
```

Comment:
Not sure how helpful this would be, as `hyprctl version` already does this. If you want pretty versions I think we could look up whether a certain ref has a name, and if it does print that instead.